### PR TITLE
{lib}[intel/2017a] mympingpong v0.8.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
{perf}[intel/2017a] mympingpong v0.8.0

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8ff95008c826209fada2c7e1653022fb for a full test report.

File: easybuild/easyconfigs/l/libxslt/libxslt-1.1.30-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxslt'
+version = '1.1.30'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxslt is the XSLT C library developed for the GNOME project
+ (but usable outside of the Gnome platform)."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libxslt/libxslt-1.1.30-intel-2017a.eb]: why build this at intel level but libxml itself at GCCcore level?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2587.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/41899db8074abf8c149d69f9b99846e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2588.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d670f9aa4652033d1ade3a9d1735266a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/15255c1c718e607704c98b498a0410d2 for a full test report.

**Pull request merged**